### PR TITLE
general styles & header styles for whole site

### PR DESCRIPTION
**Pull request by Fourbn**
should be a good enough general setup for you to work with, we can make hella pretty later!

### CODE REVIEW

File: styles/scss/_general.scss [hunk 1]
```diff
@@ -0,0 +1,25 @@
+// * {border: 1px solid red;}
+
+h1, h2, h3, p {
+   margin: 0;
+   padding: 0;
+}
+
+img {
```
Comment (dubzku) [inline styles/scss/_general.scss:8]: cool if we add a bit more to the img styles in the set-up here? I usually add the below based on a pro-tip from Esther back in webdev days to make the imgs easier to work with! 
img { 
max-width: 100%; 
height: auto; 
display: block;
}

Comment (dubzku) [inline styles/scss/_general.scss:8]: cool if we add a bit more to the img styles in the set-up here? I usually add the below based on a pro-tip from Esther back in webdev days to make the imgs easier to work with!
img {
max-width: 100%;
height: auto;
display: block;
}

**Review by dubzku — CHANGES_REQUESTED** [review]
Hey hey, I added a lil comment in on line 8 about the img styling in the setup snippet! let me know your thoughts :)

File: styles/scss/_general.scss (hunk 1 above)
Comment (Fourbn) [reply styles/scss/_general.scss:8]: yes! I was feeling that the img was missing some styles, thank you for pointing that out!

Comment (dubzku) [reply styles/scss/_general.scss:8]: No worries! Thanks for adding it in! 😄

**Comment by dubzku** [comment]
Looks great! We are READY TO STYLLLLLE this thing! 💅 💄 💁‍♀️ 💁

**Pull request merged**